hs/src/Reach/Compiler.hs

```diff
@@ -88,4 +78,8 @@ compile copts = do
       let runConnector c = (,) (conName c) <$> conGen c woutnMay pl
       crs <- HM.fromList <$> mapM runConnector connectors
       backend_js woutn crs pl
-      return ()
```

Comment:
I need this because sometimes I comment out the js generation and then the function is an error because it ends in a bind